core/arch/arm/include/tee/entry.h

```diff
@@ -25,8 +25,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef TEE_ENTRY_H
-#define TEE_ENTRY_H
+#ifndef ENTRY_H
```

Comment:
The usual pattern for inclusion guards is to take the name the file is included by (in this case `tee/entry.h`) and create a unique define based on that. That guarantees that the guard is unique.